packages/lucia-auth/src/middleware/index.ts

```diff
@@ -4,6 +4,10 @@ import type {
 	Request as ExpressRequest,
 	Response as ExpressResponse
 } from "express";
+import type {
```

Comment:
Forgot to mention.

This implementation will break whenever the type is updated. I’d recommend doing something like the SvelteKit middleware and recreating the event object only with the required properties.

Reply:
Oh you're right, I'll do it!

Reply:
Done!